src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import SharePage from "./pages/share/SharePage.js";
+import FolderPage from "./pages/folder/FolderPage.js";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<SharePage />}></Route>
+        <Route path="/share" element={<SharePage />}></Route>
```

Comment:
요구사항을 보면 share이 아닌 shared인 것 같네요~
```suggestion
        <Route path="/shared" element={<SharePage />}></Route>
```